test/github-issues/4063/issue-4063.ts

```diff
@@ -0,0 +1,42 @@
+import "reflect-metadata";
+import {createTestingConnections, closeTestingConnections, reloadTestingDatabases} from "../../utils/test-utils";
+import {expect} from "chai";
+import {Connection} from "../../../src/connection/Connection";
+
+describe("github issues > #4063 [Postgres] Query runner generates wrong PK name (with custom schema)", () => {
```

Comment:
Is it possible for this scenario to be placed under `test/functional/query-runner/drop-primary-key.ts`? & the create primary key?